pkg/config/reader.go

```diff
@@ -6,6 +6,8 @@ import (
 	"log"
 )
 
+// TODO: Escape /
```

Comment:
Is this still a todo? Or did we fix this in the code

Reply:
No just removed it